warrior/Framework/Utils/data_Utils.py

```diff
@@ -1298,10 +1300,11 @@ def get_user_specified_tag_values_in_tc(datafile, system_name, **kwargs):
     for element in kwargs:
         in_list.append(element)
     credentials = get_credentials(datafile, system_name, in_list)
-
     for element in kwargs:
         if kwargs[element] is not None:
             credentials[element] = resolve_argument_value_to_get_tag_value(datafile, system_name, kwargs[element])
+    credentials = sub_from_env_var(credentials)
```

Comment:
The correct place to handle env, data repo substitutions would be in the function resolve_argument_value_to_get_tag_value, because the current function get_user_specified_tag_values_in_tc internally uses resolve_argument_value_to_get_tag_value.

If we don't handle this support in resolve_argument_value_to_get_tag_value then users who directly use this function will not get this functionality.

I have added some comments to resolve_argument_value_to_get_tag_value  as well please incorporate them also.